packages/fast-usdc/src/delete-completed-txs.js

```diff
@@ -0,0 +1,20 @@
+// import { E } from '@endo/far';
+/* global E */
+
+/** @import {FastUSDCCorePowers} from './start-fast-usdc.core' */
+
+const trace = (...args) => console.log('FUPS', ...args);
+trace('script starting');
+
+/** @param {BootstrapPowers & FastUSDCCorePowers} powers */
+const pruneFastUsdcStorage = async powers => {
+  trace('pruneFastUsdcStorage');
+  const { fastUsdcKit } = powers.consume;
+  const { creatorFacet } = await fastUsdcKit;
+  trace(creatorFacet);
+  await E(creatorFacet).deleteCompletedTxs();
+  trace('done');
+};
+
+trace('script finishing');
+pruneFastUsdcStorage;
```

Comment:
We don't have an extension for scripts that run in CoreEval. `.core.js` is for scripts that are _built_ by a `.build.js` into a script that runs.

So I think this filename is fine for now, but it shouldn't be part of `src` because it's not a package export. How about in `scripts`?